src/Screen/Layouts/Block.php

```diff
@@ -86,6 +87,20 @@ public function description($description): self
 
         return $this;
     }
+    
+    /**
+     * Used to define block orientation.
+     *
+     * @param bool $vertical
+     *
+     * @return Block
+     */
+    public function vertical($vertical): self
```

Comment:
Let's make it possible to call this method without constantly passing the value `true`

```php
public function vertical($vertical = true)
```

Reply:
Ok, first I thought this but codeclimate scared me lol